lib/chef/compliance/runner.rb

```diff
@@ -219,7 +222,7 @@ def node_info
       end
 
       def reporter(reporter_type)
-        case reporter_type.downcase
```

Comment:
Is there a reason we don't want to downcase this anymore?

Reply:
I'd added it in the previous PR, but hadn't fully tested it in all attribute cases before we merged that.  Once I found some breakages, the faster answer was to remove it so I could wrap up the basic change.